opm/simulators/wells/BlackoilWellModelGasLift_impl.hpp

```diff
@@ -0,0 +1,297 @@
+/*
+  Copyright 2016 - 2019 SINTEF Digital, Mathematics & Cybernetics.
+  Copyright 2016 - 2018 Equinor ASA.
+  Copyright 2017 Dr. Blatt - HPC-Simulation-Software & Services
+  Copyright 2016 - 2018 Norce AS
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+// Improve IDE experience
+#ifndef OPM_BLACKOILWELLMODEL_GASLIFT_HEADER_INCLUDED
+#define OPM_BLACKOILWELLMODEL_GASLIFT_IMPL_HEADER_INCLUDED
```

Comment:
Maybe we could use the traditional include guards, akin to PR #5835, here?

Reply:
yes, already done locally, was waiting for review before push.